frontend/components/exercise/ButtonText.tsx

```diff
@@ -0,0 +1,31 @@
+interface ButtonTextProps {
```

Comment:
couple things:
- if this is the multiple selection button, name it something like "MultipleSelectionButton" otherwise it's kinda confusing
- again, do not use tailwind defaults, use arbitrary values such as px-[12px] etc. not px-3.
- remember there is text-body-semibold-cap-height, etc. you should never have to do leading or tracking for text. all text should already be in tailwind.